common/lib/xmodule/xmodule/course_module.py

```diff
@@ -830,6 +830,13 @@ class CourseFields(object):
         },
         scope=Scope.settings
     )
+    allow_unsupported_xblocks = Boolean(
+        display_name=_("Allow the creation of unsupported XBlocks"),
```

Comment:
How about one of these? @sstack22  (Glad to explain my rationale behind any of the suggestions). (Please use initial caps.)
Allow Unsupported Problem Types
Allow Unsupported Problems and Tools
Allow Unsupported Course Content Types